src/ff7_opengl.cpp

```diff
@@ -84,6 +84,9 @@ void ff7_init_hooks(struct game_obj *_game_object)
 	// Allow mouse cursor to be shown
 	replace_function(ff7_externals.dinput_createdevice_mouse, noop);
 
+  // Disable "Normal" setting in Controller section of the Config menu (it softlocks on Steam)
+  memset_code(ff7_externals.config_menu_sub + 0x8AC, 0x90, 0xE6);
```

Comment:
Same as the other PR, please move these lines where the steam edition is being checked. Thank you

Reply:
@julianxhokaxhiu ok, moved the patch to the section that checks for Steam version.